scripts/create-patch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
lets rename the file to `update-patch.sh`?

Reply:
maybe call it everywhere `patches` instead of `patch`. But I'm unsure, I would prefer the first one, wdyt?

Reply:
Renamed I think the naming is fine